Project.toml

```
@@ -18,10 +11,16 @@ Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
+[compat]
+DSP = "0.5.1, 0.6"
+FFTW = "0.2.4, 1.2"
+Reexport = "0.2.0, 0.2"
+SpecialFunctions = "0.7.1, 0.10"
```

It's not common to have a gap between two versions:

```suggestion
SpecialFunctions = "0.7.1, 0.8, 0.9, 0.10"
```